web/src/Web.App/Views/School/Index.cshtml

```diff
@@ -39,6 +39,11 @@
     @await Html.PartialAsync("_Banners", Model)
 </div>
 
+@if (Model.FederationSchools.Length > 0)
```

Comment:
query: There are 30 schools in `d01` that are alone within a federation. Should this, or an alternative partial be rendered in this case? 
```
100472
101264
101266
101290
101294
101295
101299
108426
111003
114053
114063
114064
114065
114311
114340
116715
116716
116774
116960
118318
118471
121500
121565
122641
122680
122766
133397
134171
134662
134663
```
A related question is, why are these alone?

Reply:
The logic in the front-end for me is correct and we shouldn't be adding logic to correct for the data.

This data issue is caused by the CFR data. In 2023 for some reason there are no non-lead schools in the submission.

Reply:
As a result of the work we've done on the CFR process we'll be better informed on the submission data.